sql/2020/19_Compression/mbyte_saving_of_text_compression.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+#Text Based Compression Byte Savings
+SELECT 
+  _TABLE_SUFFIX AS client,
+  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.details.overallSavingsBytes")as INT64)/1024/1024) mbyte_savings,
+  COUNT(0) num_pages,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct_pages       
+FROM 
+  `httparchive.lighthouse.2020_08_01_*`
+WHERE
+  JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.score") != "1"
```

Comment:
I'd suggest removing this so we can see the % of pages that don't have any savings.

Reply:
agree to make a change of suggestion.